public/views/projects-migrated.html

```diff
@@ -39,10 +33,22 @@ <h4 class="site-header">{{migratedProject.site_name}}</h4>
 				 <span ng-show="migratedProject.destination_url===null">Exported to <span ng-if="migratedProject.destination_type==='mailarchivezip'">zip</span><span ng-if="migratedProject.destination_type!=='mailarchivezip'">{{migratedProject.destination_type}}</span></span>
          <span ng-show="migratedProject.migrated_by"> by {{migratedProject.migrated_by}}</span>
 				</div>
-        <div class="col-md-6 col-lg-6 col-sm-6 col-xs-6" style="text-align:right">
-          <small ng-show="migratedProject.status.data.counts.failures !==0" class="export-fail">
-            There were {{migratedProject.status.data.counts.failures}} failures
+        <div class="col-md-6 col-lg-6 col-sm-6 col-xs-6 {{migratedProject.status.status}}" style="text-align:right">
+          <small ng-show="migratedProject.status.counts.successes ===0 && migratedProject.status.counts.errors ===0">
```

Comment:
Usually you will see the partial success in case of Mbox and google migration. Error is when a email is not migrated. but if attachments are dropped, still email is migrated then this is not a error case `Status` = PARTIAL, you will see partial success in the Items []. If a email is not migrated and some emails dropped attachments(partial success) this will be ERROR as one email did not do migrated